ab2d-aggregator/src/main/java/gov/cms/ab2d/aggregator/FileOutputType.java

```diff
@@ -0,0 +1,38 @@
+package gov.cms.ab2d.aggregator;
+
+import java.io.File;
+
+/**
+ * Taken from AB2D. Describes the different file endings for the data and error files created by the aggregator
+ */
+public enum FileOutputType {
+    NDJSON(".ndjson"),
+    NDJSON_ERROR("_error.ndjson");
+
+    private final String suffix;
+
+    FileOutputType(String suffix) {
+        this.suffix = suffix;
+    }
+
+    public String getSuffix() {
+        return suffix;
+    }
+
+    public static FileOutputType getFileType(File file) {
+        return getFileType(file.getAbsolutePath());
+    }
+
+    public static FileOutputType getFileType(String file) {
+        if (file == null) {
```

Comment:
Can this be in the form of an assert instead?  It doesn't make sense to me to allow for null file references for determining its type.  I would rather just throw a NPE if someone asks for information on a null.

Reply:
I'll have it return UNKNOWN but I'm not a big fan of assert used that way.